.python_package/setup.py

```diff
@@ -0,0 +1,27 @@
+# GridTools
+#
+# Copyright (c) 2014-2021, ETH Zurich
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+
+from pkg_resources import parse_version
+from setuptools import find_packages
+from skbuild import setup
+
+if __name__ == "__main__":
+    version_file = pathlib.Path(__file__).absolute().parent.parent / "version.txt"
+    setup(
+        package_dir={"": "py_src"},
+        packages=find_packages(where="py_src"),
+        version=str(parse_version(version_file.read_text())),
+        include_package_data=True,
+        package_data={"gridtools": ["*.hpp", "*.h", "*.cmake"]},
```

Comment:
According to the (arguably confusing) configuration for packaging data files in python packages, using the `package_data` option in `setup.cfg` makes `MANIFEST.in` and `include_package_data` redundant. Since the behavior heavily depends on the `setuptools` version I would only use `package_data` and make sure the build system in `project.toml` uses a modern  `setuptools` version  (check more info here: https://github.com/abravalheri/experiment-setuptools-package-data#doc2)

Reply:
I will test if manifest etc is really necessary. I suspect it isn't.

Reply:
It was unnecessary, as is `setup.py` with the recent changes.